File: src/main/java/br/uff/tecnomarias/domain/entity/Feedback.java
package br.uff.tecnomarias.domain.entity;

import javax.persistence.*;
import javax.validation.constraints.NotNull;
import java.io.Serializable;

@Entity
public class Feedback implements Serializable {
    private static final long serialVersionUID = 1L;

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    @NotNull
    @Column(length = 500, nullable = false)
    private String comentario;

    @OneToOne
    private PessoaFisica pessoa;

    public Feedback() {
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getComentario() {
        return comentario;
    }

    public void setComentario(String comentario) {
        this.comentario = comentario;
    }

    public PessoaFisica getPessoa() {
        return pessoa;
    }

    public void setPessoa(PessoaFisica pessoa) {
        this.pessoa = pessoa;
    }
}

File: src/models/entities/Feedback.js
export class Feedback {
    constructor(id, comentario, idPessoa) {
        this._id = id;
        this._comentario = comentario;
        this._idPessoa = idPessoa;
    }

    get id() {
        return this._id;
    }

    set id(id) {
        this._id = id;
    }

    get comentario() {
        return this._comentario;
    }

    set comentario(comentario) {
        this._comentario = comentario;
    }

    get idPessoa() {
        return this._idPessoa;
    }

    set idPessoa(idPessoa) {
        this._idPessoa = idPessoa;
    }
}